e2e-tests/content-strings/pages/insurance/index.js

```diff
@@ -104,8 +104,32 @@ const START = {
   },
   BODY_1: "Depending on your export contract, you may also need your buyer's annual report and accounts.",
   BODY_2: 'You do not need to complete all answers in one session.',
-  BODY_3: "You'll usually get a decision back from UKEF within 2 weeks. This is because underwriters need to carry out checks on risks around the buyer.",
-  BODY_4: 'If you need it more urgently, you can tell us before you submit your application.',
+  BODY_3: "You'll usually get a decision back within 7 working days, if we do not need to ask you any further questions.",
+  BODY_4: 'If we need to gather extra information, it may take around 2 to 3 weeks.',
+  SIGN_IN: {
+    BODY_1: 'You can',
+    LINK: {
+      TEXT: 'sign in to your account',
+      HREF: ROUTES.INSURANCE.ACCOUNT.SIGN_IN.ROOT,
+    },
+    BODY_2: "to continue an application, if you've already started one.",
+  },
+  FIND_OUT_MORE: {
+    BODY_1: 'You can',
+    LINK: {
+      TEXT: 'find out more about UKEF export insurance',
+      HREF: LINKS.EXTERNAL.GUIDANCE,
+    },
+    BODY_2: ", if you want to know who's eligible and what's covered.",
+  },
+  QUOTE: {
+    BODY_1: 'You can also',
```

Comment:
Could we be more descriptive instead of BODY_1/2 etc? e.g `YOU_CAN`, `TO_CONTINUE`

Also don't think we should a content string with a comma